src/WebJobs.Script/Host/ScriptHost.cs

```diff
@@ -408,13 +465,55 @@ protected virtual void Initialize()
                     hostConfig.StorageConnectionString = null;
                 }
 
+                var serverImpl = new FunctionRpcImpl(EventManager);
```

Comment:
To keep this method more readable, recommend factoring all this language worker init into its own method